relay-server/src/processing/spans/mod.rs

```diff
@@ -139,11 +144,10 @@ impl processing::Processor for SpansProcessor {
 
         process::normalize(&mut spans, &self.geo_lookup);
         filter::filter(&mut spans, ctx);
+        process::scrub(&mut spans, ctx);
```

Comment:
In the error processing pipeline, we [enforce quotas before scrubbing](https://github.com/getsentry/relay/blob/1417935c98a56c340f6171ef9af69b0d1898eb79/relay-server/src/services/processor.rs#L1695-L1700) (that's probably also why the TODO was below `enforce_quotas`).

It shouldn't matter much in terms of performance because the bulk of rate limited events are dropped on the fast path, not here, but maybe we should keep the order for consistency.

Reply:
Moved scrubbing after quota enforcement for both logs and spans.